msg/HttpRequest.msg

```diff
@@ -0,0 +1,28 @@
+# HTTP request message
+
+# Standard ROS header
+std_msgs/Header header
```

Comment:
one thing internally we need to follow up on is whether we want to align on using headers in our message definitions or just a timestamp. I'm starting to lean towards the latter, but this is fine for now